next/platform/ios/ios.cmake

```diff
@@ -89,4 +92,89 @@ target_link_libraries(
         z
 )
 
+if(IOS_RENDER_TESTING)
```

Comment:
`CMAKE_SYSTEM_NAME STREQUAL iOS`?

Reply:
Currently we have another job next-ios-xcode-release, which also has ios platform.